src/components/Form/Form.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import {Button} from '../Button';
+import {MultiSelect} from '../MultiSelect/';
+
+import styles from './Form.module.css';
+
+interface FormProps {
+    initialValues?: FormValues;
+}
+
+interface FormValues {
+    title: string;
+    date: string;
+    url: string;
+    genre: string[];
+    overview: string;
+    runtime: string;
+}
+
+export const Form = ({
+    initialValues
+}: FormProps) => {
+    const [formData, setFormData] = React.useState<FormValues>(initialValues ?? {} as FormValues);
```

Comment:
This is not a good practice to relate on props in your state. You can pass `values` and `callbacks` from upper component instead of relation on props

Reply:
This is done